frontend/language/src/nb.json

```
@@ -910,6 +910,7 @@
   "schema_editor.promote": "Konverter til type",
   "schema_editor.properties": "Egenskaper",
   "schema_editor.reference_to": "Refererer til",
+  "schema_editor.reference_type_in_use_title": "Du kan ikke dra samme type som er allerede brukt i referense.",
```

I find this a little unclear. How about something like this? 😊 
```suggestion
  "schema_editor.reference_type_in_use_title": "Typer kan ikke referere til seg selv.",
```

---

KK = Klart og Kort ❤️ Jeg skal fekse 😊